modules/examples/todolist/src/main/scala/todo/runtime/implicits.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package todo
+package runtime
+
+import java.util.Properties
+
+import cats._
+import cats.effect._
+import com.twitter.util._
+import com.zaxxer.hikari.{HikariConfig, HikariDataSource}
+import doobie.hikari._, doobie.hikari.implicits._
+import doobie._, doobie.implicits._
```

Comment:
Maybe we could split these import statements into different lines, for consistency